style1.css

```diff
@@ -390,3 +390,285 @@ textarea::placeholder {
 .cont {
   margin: 20px 0;
 }
+
+@media only screen and (min-width: 768px) {
+  #navigation {
+    display: flex;
+    justify-content: space-between;
+    padding: 16px 66px;
+    box-shadow: 0 1px 10px -5px grey;
+    position: fixed;
+    height: 72px;
+  }
+
+  nav {
+    display: flex;
+    justify-content: flex-end;
+  }
+
+  .menu-bar {
+    display: none;
+  }
+
+  .nav-list {
+    margin-left: initial;
+    display: flex;
+    justify-content: flex-end;
+    align-items: center;
+    gap: 10px;
+    list-style: none;
+    text-transform: capitalize;
+  }
+
+  .nav-list .nav-content a {
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    color: #344563;
+  }
+
+  .nav-list .nav-content a:hover {
+    text-decoration: underline;
+    color: #6070ff;
+    transition: 0.1s;
+    font-weight: bold;
+  }
+
+  .hero-section {
+    background-image: url(/images1/Header-bg.svg);
```

Comment:
- [ ] Your images for all icons and backgrounds both for mobile and desktop screens aren't visible. Kindly fix the paths using relative paths. Also, ensure you don't have horizontal scrolling in both cases(mobile and desktop): 

> ![image](https://user-images.githubusercontent.com/91486791/236466679-0a2d6dd4-afe9-41d5-895d-dce110c61046.png)

Reply:
- [ ] Corrected :white_check_mark: